.github/workflows/audit.yml

```diff
@@ -25,6 +25,7 @@ jobs:
     - uses: actions/checkout@v3
       with:
           submodules: true
+          persist-credentials: true
```

Comment:
why do we need this?

Reply:
just follow the error [message](https://github.com/crypto-org-chain/chain-main/actions/runs/11357568574/job/31590653692), previous ci was failed

Reply:
https://github.com/actions/checkout#usage
the doc suggests it defaults to true, maybe try to upgrade to v4?